controller/action/AcquisitionActions.class.php

```diff
@@ -59,6 +59,11 @@ public static function executeVerify(string $token)
         return ['acquisition/verify', ['token' => $token]];
     }
 
+    public static function executeAutoVerify()
```

Comment:
FYI, you could have omitted a route definition and controller function entirely if you had just put the template as `page/verify` (there is a catch all route for this pattern to avoid writing empty controllers). However, in this case, since one would reasonably expect a controller to exist, I think it's fine/good to have the route defined.